Formula/corectl.rb

```diff
@@ -7,13 +7,14 @@ class Corectl < Formula
 
   bottle do
     cellar :any
+    rebuild 1
```

Comment:
Definitely not happy about it but if Scotland gets independence it may be less painful. Thanks!